lib/datasource.js

```diff
@@ -844,6 +844,16 @@ DataSource.prototype.deleteModelByName = function(modelName) {
   delete this.connector._models[modelName];
 };
 
+/**
+ * Remove all models from the registry, but keep the connector instance
+ * (including the pool of database connections).
+ */
+DataSource.prototype.reset = function() {
```

Comment:
Let's rename it to `deleteAllModels` as `reset` hints the connection pool will be reset too.

Reply:
Yeah, I was aware of that possibility. In the future, we may want to add more steps than just deleting all models, that's why I wanted to use a more generic name than `deleteAllModels`. 

Let's follow YAGNI principle and don't worry about things we may need in the future.

I am going to rename the method to `deleteAllModels` as suggested.